src/Administration/Resources/app/administration/src/module/sw-settings-custom-field/snippet/en.json

```
@@ -80,7 +80,10 @@
         "tooltipAllowCartExpose": "Activate, to make this custom field available in the payload of shopping cart line items. This is required in order to initially use this custom field in the Rule Builder or custom templates.",
         "tooltipCartExposeDisabled": "This setting can currently not be changed, because the custom field is in use by one or more conditions in the Rule Builder.",
         "labelStoreApiAware": "Visible in Store API",
-        "tooltipStoreApiAware": "When active, this field will be available in the Store API. This option can be disabled to store custom data only internally and not make it available via the Store API."
+        "tooltipStoreApiAware": "When active, this field will be available in the Store API. This option can be disabled to store custom data only internally and not make it available via the Store API.",
+        "labelAllowSearchable": "Searchable",
+        "tooltipAllowSearchable": "When active, this custom field will be indexed and can be used in search configurations. Disable this option to reduce index storage size if the field does not need to be searchable.",
+        "bannerSearchableEnabled": "After changing the searchable status, please reindex your data or update products manually to ensure accurate search results."
```

```suggestion
        "labelAllowSearchable": "Include in search",
        "tooltipAllowSearchable": "When active, this custom field will be indexed and can be used in search configurations. If this field is not required for the search, it should be disabled to reduce the index storage size  and improve the performance of the search.",
        "bannerSearchableEnabled": "It is recommended to reindex the data, if the search status has changed, to ensure accurate search results."
```

I would suggest to change the English description as well. I think "Searchable" doesn't really fit in this context. Also the tooltip was in my opinion a bit too technical and should also explain the benefit of deactivating it a bit better for someone who is not too technical.
Also removed the "update products manually" part from the banner. I don't think, this is really a feasible suggestion and no one would probably do that for all product manually